src/zjs_i2c.json

```diff
@@ -2,6 +2,7 @@
     "module": "i2c",
     "require": "i2c",
     "depends": ["buffer", "i2c_a101", "i2c_k64f"],
+    "targets": ["arduino_101", "frdm_k64f"],
```

Comment:
@nagineni  Sorry I'm so late in getting to see this, but what does this I2C file have to do with ashell? Does I2C not work on ashell with these other boards?

Reply:
@brianjjones Currently I2C module is buildable only for a101 [1] and frdm_k64f [2] targets. When we build IDE for other boards, the build fails with i2c related errors since we haven't tested and enabled the module for other boards (e.g olimex stm32-e407, stm32f407g_disco)

[1] https://github.com/01org/zephyr.js/blob/master/src/zjs_i2c_a101.json#L4
[2] https://github.com/01org/zephyr.js/blob/master/src/zjs_i2c_k64f.json#L3

Reply:
Makes sense thanks for the clarification.